docs/src/main/asciidoc/_project-features-wiremock.adoc

```diff
@@ -63,6 +63,12 @@ NOTE: Actually, WireMock always loads mappings from `src/test/resources/mappings
 well as* the custom locations in the `stubs` attribute. To change this behavior, you can
 also specify a files root, as described in the next section of this document.
 
+NOTE: Also, the mappings in the `stubs` location are not considered part of Wiremock's "default mappings" and calls
+to `com.github.tomakehurst.wiremock.client.WireMock.resetToDefaultMappings` during a test will not result in the mappings
+in the `stubs` location to be included. However, the `org.springframework.cloud.contract.wiremock.WireMockTestExecutionListener`
```

Comment:
Is marking up w/ "` `" sufficient for these class names or should we link to the code?

Reply:
IMO it's fine.